packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [35.1.0]
+
+### Added
+
+- Added optional `topBid` type in `NftMetadata` ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Added optional `floorAsk` in `TokenCollection` type ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Added linea mainnet support to nft detection supported networks ([#4515](https://github.com/MetaMask/core/pull/4515))
+- Added a batch call to get collections API in detectNfts function ([#4443](https://github.com/MetaMask/core/pull/4443))
+- Added a call to get collections API inside getNftInformationFromApi ([#4443](https://github.com/MetaMask/core/pull/4443))
+
+### Changed
+
+- Changed attributes type in `NftMetadata` to `Attributes[]` ([#4522](https://github.com/MetaMask/core/pull/4522))
```

Comment:
nit: Changelog entries should start with a present-tense verb (https://common-changelog.org/#241-change)

```suggestion
- Add optional `topBid` type in `NftMetadata` ([#4522](https://github.com/MetaMask/core/pull/4522))
- Add optional `floorAsk` in `TokenCollection` type ([#4522](https://github.com/MetaMask/core/pull/4522))
- Add linea mainnet support to nft detection supported networks ([#4515](https://github.com/MetaMask/core/pull/4515))
- Add a batch call to get collections API in detectNfts function ([#4443](https://github.com/MetaMask/core/pull/4443))
- Add a call to get collections API inside getNftInformationFromApi ([#4443](https://github.com/MetaMask/core/pull/4443))

### Changed

- Change attributes type in `NftMetadata` to `Attributes[]` ([#4522](https://github.com/MetaMask/core/pull/4522))
```